src/Weapsy/appsettings.json

```diff
@@ -3,7 +3,7 @@
     "InstrumentationKey": ""
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=.\\SQLExpress;Initial Catalog=weapsy.dev;Integrated Security=SSPI"
+    "DefaultConnection": "Data Source=localhost;Initial Catalog=weapsy.dev;user id=sa;password=111;Integrated Security=SSPI"
```

Comment:
Actually I updated the connection string to make the application run at my local. Do I have to add `appsettings.json` in `.gitgnore`? Please let me know.
However, I have reverted it back to the original to read `SQLExpress` for now.